### DISCUSSION TITLE
Koreader for Pocketbook Model PB741 (PocketBook Inkpad Color)

### DISCUSSION THREAD

**Discussion by shubhrode** [koreader/koreader#7849] [Q&A]
I tried running the pocketbook version of it on my new Inkpad Color, and it didn't execute successfully. (see image). Any idea if that's something fixable at my end or I will have to wait for that version to be included in a future update?
[http://imgur.com/gallery/4T4A4BA](url)

**Comment by Frenzie** [comment]
If you grab the most nightly from http://build.koreader.rocks/download/nightly/ it should load (without color).

**Reply by shubhrode** [reply]
Thanks a lot! It works... even greyscale is great.
In fact greyscale only is faster for comics, even the ones which are already in greyscale load faster on koreader!

**Reply by SoulMist17** [reply]
Any Updates on further Inkpad Color support?

**Reply by NiLuJe** [reply]
If you complain loudly enough to PocketBook to actually get up to date kernel sources, we might be able to just offload it to the kernel.

Other than that, the DIY approach is https://github.com/koreader/koreader-base/pull/1191

TL;DR: We don't have a PB maintainer, so, no.
